Sources/StreamChat/Controllers/ReactionListController/ReactionListController.swift

```diff
@@ -172,7 +172,13 @@ extension ChatReactionListController {
             _ itemCreator: @escaping (MessageReactionDTO) throws -> ChatMessageReaction
         )
             -> ListDatabaseObserverWrapper<ChatMessageReaction, MessageReactionDTO> = {
-                ListDatabaseObserverWrapper(isBackground: $0, database: $1, fetchRequest: $2, itemCreator: $3)
+                ListDatabaseObserverWrapper(
+                    isBackground: $0,
+                    database: $1,
+                    fetchRequest: $2,
+                    itemCreator: $3,
+                    itemReuseKeyPaths: nil // no id
```

Comment:
What would be the behaviour for reactions in this case? All of the models would be re-created, right?

Reply:
Yes, everything in FRC is recreated. 
Did not have a good idea how to create a reliable reuse key here, therefore skipped for now.

Reply:
MessageReactionDTO has `id`, I'll expose it on the ChatMessageReaction level and then I can use it for reusing.

Reply:
Done